### PR TITLE
Update changelog and readme file

### PR DESCRIPTION
**Pull request by dafigerdjikova**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,90 @@
...
+- Added reference to mt core service and MT core account management url is requiered;
+
+```json
+  "mtCoreAccountsMgmtServiceUrl": "mtcore account url",
+```
+
+- Added reference to chest service and chest url is requiered;
+
+```json
+  "chestUrl": "http://chest.mt.svc.cluster.local:5011",
```
Comment (iliyan-georgiev-ph) [inline CHANGELOG.md]: Should this URL be here considering in every other place it's been omitted

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,90 @@
+## 2.11.0 (March 8, 2019)
```
Comment (iliyan-georgiev-ph) [inline CHANGELOG.md]: The new secrets don't seem to be included here

Comment (dafigerdjikova) [reply CHANGELOG.md]: SO I don't have them :D Are they added here
https://lykke-snow.atlassian.net/wiki/spaces/LEGO/pages/511901697/User+Secrets
Please add them in confluence and I will update the changelog

Comment (iliyan-georgiev-ph) [reply CHANGELOG.md]: It's just the Redis connection string, that's already added to the list

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,90 @@
...
+      ]
+    }
+  ]
+```
+
+-Added activity publisher settings, new exchange for login events is required;
+
+```json
+"ActivityPublisherSettings": {
+    "ConnectionString": "correct connection string",
```
Comment (Waqas-Zamir) [inline CHANGELOG.md]: btw we should move this to `ConnectionStrings:RabbitMq` to align with the other services

File: CHANGELOG.md (hunk 2 above)
Comment (dafigerdjikova) [reply CHANGELOG.md]: So it is from the previous release correct?

File: CHANGELOG.md (hunk 3 above)
Comment (dafigerdjikova) [reply CHANGELOG.md]: I will update it

File: README.md [hunk 4]
```diff
@@ -68,6 +69,9 @@ These available variables are detailed below:
   | SessionConfig:* | Session timeout in seconds, default value will be 300 seconds |
   | mtCoreAccountsMgmtServiceUrl | Url for MT Core accounts management service |
   | SecurityGroups:* | Role permissions mapping
+  | ActivityPublisherSettings:* | Activity publisher settings
+  | chestUrl | Url for Chest service |
+  | ConnectionStrings | RabbitMQ connection string
```
Comment (Waqas-Zamir) [inline README.md]: `ConnectionStrings:RabbitMq`

File: README.md [hunk 5]
```diff
@@ -68,6 +69,9 @@ These available variables are detailed below:
   | SessionConfig:* | Session timeout in seconds, default value will be 300 seconds |
   | mtCoreAccountsMgmtServiceUrl | Url for MT Core accounts management service |
   | SecurityGroups:* | Role permissions mapping
+  | ActivityPublisherSettings:* | Activity publisher settings
```
Comment (Waqas-Zamir) [inline README.md]: better to mention sub properties
`ActivityPublisherSettings:ExchangeName`
`ActivityPublisherSettings:IsDurable`

Comment (dafigerdjikova) [reply README.md]: @Waqas-Zamir please check I updated the previous sections as well

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,110 @@
...
+-Added activity publisher settings, new exchange for login events is required;
+
+```json
+"ActivityPublisherSettings": {
+    "ExchangeName": "lykke.axle.activities",
+    "IsDurable": true
+  }
+```
+
+- Added reference to mt core service and MT core account management url is requiered;
```
Comment (benoitbousquet-lkk) [inline CHANGELOG.md]: "requiered" typo

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,110 @@
...
+
+```json
+  "mtCoreAccountsMgmtServiceUrl": "mtcore account url",
+```
+
+- Added reference to chest service and chest url is requiered;
```
Comment (benoitbousquet-lkk) [inline CHANGELOG.md]: "requiered" typo

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,110 @@
...
+
+#### Secrets variables
+
+  | ConnectionStrings:RabbitMq / RABBITMQ_CONNECTIONSTRING | Connection string to RabbitMq server |
+
+All latest configuration changes that are used and working for dev environment can be found in ```appSettings.json```
+
+### Axle.Client
+
+Axle client which can be used by otehr services in order to call endpoint from axle service
```
Comment (benoitbousquet-lkk) [inline CHANGELOG.md]: "otehr" typo

**Pull request merged**